### ISSUE TITLE
Install instructions

### ISSUE THREAD

**Issue by seba-1511** [adamvoss/languagetool-languageserver#1]
This project looks quite promising, especially given that most editors are including support for LSP. 

Could you provide install instructions for people that are not familiar with gradle/the java ecosystem ?

Thanks,
Séb

**Comment by adamvoss** [comment]
Are you looking for build instructions or installation instructions?

Build instructions I can do:  `gradle build`

Installation instructions would become a matter of what you mean by "install."  Currently the only client I know of is [**vscode-languagetool**](https://github.com/adamvoss/vscode-languagetool) which bundles this server so separate installation is not required.

**Comment by seba-1511** [comment]
Thanks for your quick reply. 

I managed to run `gradle build`, but how would I start the language server  ? (e.g. for python I would start it with `pyls` , for C `clangd`, etc...)

My specific language client is for [neovim](https://github.com/autozimu/LanguageClient-neovim), but I assume this would be useful for others too.

**Comment by adamvoss** [comment]
Here is how you can start this application:

```bash
gradle installDist
./build/install/languagetool-languageserver/bin/languagetool-languageserver $PORT_NUMBER
```

Where `$PORT_NUMBER` is the port for the socket connection (opened by the client) that the server will communicate with.
___

I'll be happy to accept pull requests that add better support for other clients, but this is not something I will be undertaking myself in the foreseeable future.  This does use `LSP4J` and speaks the protocol, but the actual communication piece was only written enough to support the VSCode client I was writing at the time.

**Comment by plexigras** [comment]
@seba-1511 did you get it to work?

**Comment by seba-1511** [comment]
No, didn't have the time to fiddle and try to connect it with neovim-language-client.

**Comment by davidlday** [comment]
Please see issue #6.
